src/test.mediacreation.cpp

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 The Foundry Visionmongers Ltd
+
+// Include all headers to test they are where we expect and can be
+// compiled.
+#include <openassetio/TraitsData.hpp>
+
+#include <openassetio_mediacreation/openassetio_mediacreation.hpp>
+#include <openassetio_mediacreation/traits/content.hpp>
+#include <openassetio_mediacreation/traits/content/LocatableContentTrait.hpp>
+#include <openassetio_mediacreation/traits/managementPolicy.hpp>
+#include <openassetio_mediacreation/traits/managementPolicy/ManagedTrait.hpp>
+#include <openassetio_mediacreation/traits/managementPolicy/ResolvesFutureEntitiesTrait.hpp>
+#include <openassetio_mediacreation/traits/timeline.hpp>
+#include <openassetio_mediacreation/traits/timeline/ClipTrait.hpp>
+#include <openassetio_mediacreation/traits/timeline/TimelineTrait.hpp>
+#include <openassetio_mediacreation/traits/timeline/TrackTrait.hpp>
+#include <openassetio_mediacreation/traits/traits.hpp>
+
+using namespace openassetio_mediacreation;
```

Comment:
[consider][minor] Not really best practice, and doesn't save much typing anyway.

Reply:
Fair enough, it was just to deal with an awkward line wrap because of the character limits. Don't mind either way.

[b58f98e](https://github.com/OpenAssetIO/OpenAssetIO-Test-CMake/pull/7/commits/b58f98e413bbe420c68347d4fb7fd7614561e66e)